enterprise/releases/2023-07-0.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: '2023.07.0 Release'
+backlink_href: /docs/enterprise/releases
+backlink_title: 'pganalyze Enterprise Server: Release Changelog'
+---
+
+**Docker tag:** `quay.io/pganalyze/enterprise:v2023.07.0`
+
+## Notes
+
+- This release requires a database migration when coming from older releases, be sure to follow the [recommended upgrade steps](/docs/enterprise/upgrade)
+
+## Features
+
+* Index Advisor
+  - Support analyzing queries that require the "ltree" extension <!-- PR 2913 -->
+  - Add support for IMMUTABLE functions <!-- PR 2937 -->
+    - This fixes errors like "functions in index expression must be marked IMMUTABLE" or "functions in partition key expression must be marked IMMUTABLE".
+  - Fix query analysis of INSERT INTO SELECT by ensuring correct column order for query analysis <!-- PR 2916 -->
+    - This fixes type coercion errors like "column "X" is of type A but expression is of type B" on specific cases of INSERT INTO SELECT statements that relied on the column positions of the target table.
+  - Support foreign key constraints that reference tables with indexes using custom functions <!-- PR 2915 -->
```

Comment:
Uhmm, this sentence is confusing. Original change log:

> Fix analysis error when foreign key constraint tables have indexes using custom functions

How about...

> Support indexes using custom functions in foreign key constraint reference tables

Reply:
Reworded to

> Support indexes using custom functions in tables referenced via foreign key constraints